packages/atlas/src/providers/wallet/wallet.provider.tsx

```diff
@@ -84,6 +128,36 @@ export const WalletProvider: FC<PropsWithChildren> = ({ children }) => {
     },
     [setWallet, setWalletAccounts, setWalletStatus]
   )
+  const signInWithWalletConnect = useCallback(async () => {
+    const walletConnectParams = await getWalletConnectParams()
+    const wcWallet = new WalletConnectWallet(walletConnectParams, atlasConfig.general.appName)
```

Comment:
Should we even try to initialize WC wallet if there might not be a project ID?

`projectId: walletConnectConfig.walletConnectProjectId || '',`